core/tasks/get_data.py

```diff
@@ -72,8 +72,9 @@ def fetch_deep_data():
         if afs_dict is None:
             logger.warning("Could not fetch analysis frameowrks. Exiting.")
             return
-        for prj in ToFetchProject.objects.filter(
-            status=ToFetchProject.FetchStatus.NOT_FETCHED,
+        for prj in (
+            ToFetchProject.objects.filter(status=ToFetchProject.FetchStatus.NOT_FETCHED)
```

Comment:
It's better to use `status__in=[ToFetchProject.FetchStatus.NOT_FETCHED, ToFetchProject.FetchStatus.FETCHIGN]` instead of using union.